projects/suricata/build.sh

```diff
@@ -45,6 +45,7 @@ make install
 cd ..
 
 export CARGO_BUILD_TARGET="x86_64-unknown-linux-gnu"
+rustup component add rust-src --toolchain nightly
```

Comment:
You may want to do this in the Docker image, so that it works for other Rust projects

Reply:
Can you submit a patch to https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/Dockerfile#L79 with a comment on why it is needed. Thanks!

Reply:
It comes from this discussion :
https://github.com/google/oss-fuzz/issues/3469#issuecomment-648463139

We need to have `-Z build-std` to rebuild Rust standard library with MSAN

And to have this to work, we need a few things including this `rustup component add rust-src` cf 
https://doc.rust-lang.org/cargo/reference/unstable.html#build-std